doc/index.rst

```diff
@@ -0,0 +1,17 @@
+.. _index:
+
+Welcome to npmx zephyr PMIC support repository!
+###############################################
+
+:ref:`npmx-zephyr <index>` is a repository with zephyr-like samples for Nordic Semiconductor `npmx library`_ and driver integration with `zephyr`_ environment.
```

Comment:
```suggestion
:ref:`npmx-zephyr <index>` is a repository showcasing the integration of `npmx library`_ into an RTOS environment, using `Zephyr`_ RTOS as an example.
```
Is it ok to use `Zephyr`? Will the link still work?

Reply:
`Zephyr` works good